static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -441,19 +482,63 @@ function AutofixRootCauseDisplay({
             >
               {t('Give Solution')}
             </Button>
-            <Button
-              size="sm"
-              priority={
-                rootCauseSelection && 'cause_id' in rootCauseSelection
-                  ? 'default'
-                  : 'primary'
-              }
-              busy={isSelectingRootCause}
-              onClick={handleSelectRootCause}
-              title={t('Let Seer plan a solution to this issue')}
-            >
-              {t('Find Solution')}
-            </Button>
+            {cursorIntegration ? (
+              <ButtonBar merged gap="0">
+                <Button
+                  size="sm"
+                  priority={
+                    rootCauseSelection && 'cause_id' in rootCauseSelection
+                      ? 'default'
+                      : 'primary'
+                  }
+                  busy={isSelectingRootCause}
+                  disabled={isLoadingAgents}
+                  onClick={handleSelectRootCause}
+                  title={t('Let Seer plan a solution to this issue')}
+                >
+                  {t('Find Solution')}
+                </Button>
+                <DropdownMenu
+                  items={[
+                    {
+                      key: 'cursor-agent',
+                      label: t('Send to Cursor Background Agent'),
+                      onAction: handleLaunchCursorAgent,
+                      disabled: isLoadingAgents || isLaunchingAgent,
+                    },
+                  ]}
+                  trigger={(triggerProps: any, isOpen: boolean) => (
```

Comment:
these should have types already
```suggestion
                  trigger={(triggerProps, isOpen) => (
```

Reply:
it might type error if you aren't using DropdownButton